lib/index.js

```diff
@@ -9,20 +9,13 @@ import Timer from './plugins/timer';
 import Touch from './plugins/touch';
 import a11y from './plugins/a11y';
 
-export let shower; // eslint-disable-line
-
 const dataAttrsOptions = [
     'debug-mode',
     'slides-selector',
 ];
 
 const hasOptions = typeof showerOptions !== 'undefined';
 const options = hasOptions ? showerOptions : {};
-const containerSelector = options.shower_selector || defaultOptions.container_selector;
-const container = document.querySelector(containerSelector);
-if (!container) {
-    throw new Error(`Shower container with selector ${containerSelector} not found.`);
-}
 
 const autoInit = 'auto_init' in options ? options.auto_init : true;
```

Comment:
Do we need auto init feature anymore?

Reply:
It might be a useful feature to have, though I don’t see any strong use cases for now, I’d keep it untill the next refactoring :)